src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -194,19 +410,23 @@ namespace nbl::ext::imgui
 
 				if (!spirv)
 				{
-					m_creationParams.utilities->getLogger()->log("Could not compile \"%s\" shader!", system::ILogger::ELL_ERROR, key.value);
+					creationParams.utilities->getLogger()->log("Could not compile \"%s\" shader!", ILogger::ELL_ERROR, key.value);
 					return nullptr;
 				}
 
-				auto gpu = m_creationParams.utilities->getLogicalDevice()->createShader(spirv.get());
+				auto gpu = creationParams.utilities->getLogicalDevice()->createShader(spirv.get());
 
 				if (!gpu)
-					m_creationParams.utilities->getLogger()->log("Could not create GPU shader for \"%s\"!", system::ILogger::ELL_ERROR, key.value);
+					creationParams.utilities->getLogger()->log("Could not create GPU shader for \"%s\"!", ILogger::ELL_ERROR, key.value);
 
 				return gpu;
 			};
 
-			system->mount(smart_refctd_ptr(archive), NBL_ARCHIVE_ALIAS.data());
+			//! we assume user has all Nabla builtins mounted - we don't check it at release
+			assert(system->areBuiltinsMounted());
```

Comment:
I'd rather not crash, also I'm not sure what our `ISystem` does if you try to mount two archives with same alias

Reply:
> I'd rather not crash due to assert

resolved